enterprise/server/cmd/ci_runner/main.go

```diff
@@ -2521,8 +2521,13 @@ func runBazelWrapper() error {
 		return err
 	}
 
+	// Get the current bazel workspace path where we expect to find the
+	// workspace rc file. If this fails just treat it like we aren't in a
+	// workspace.
+	workspacePath, _ := currentBazelWorkspaceAbsPath()
```

Comment:
Should we at least log the error to the executor?

Reply:
Changed it to just return an error - if there's an error here other than NotFound then it's probably some sort of filesystem error, and it shouldn't matter much whether we return it here or if we let bazel run and return the error